0.3/Dockerfile

```diff
@@ -16,14 +16,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM openjdk:8-jdk as builder
+FROM debian:buster as builder
```

Comment:
For even more efficiency, you can maybe use `debian:buster-slim` for the build step, if it has everything you need.